lxd/instance/drivers/driver_qemu.go

```diff
@@ -2044,29 +2044,34 @@ func (d *qemu) setupNvram() error {
 }
 
 func (d *qemu) qemuArchConfig(arch int) (path string, bus string, err error) {
+	basePath := ""
+	if shared.InSnap() && os.Getenv("SNAP_EXT_QEMU_PREFIX") != "" {
```

Comment:
We tend to use noun_adjective format for most things, so could this be SNAP_QEMU_EXT_PREFIX?

Or shall we just drop the EXT bit and use SNAP_QEMU_PREFIX (the external part being implicit if the env var is populated)?

Reply:
Have changed this. Thanks!